extensions/2.0/Vendor/DRACO_animation_compression/README.md

```diff
@@ -0,0 +1,236 @@
+        {
+            "componentType" : 5123,
+            "count" : 100,
+            "type" : "SCALAR"
+        },
+        {
+            "componentType" : 5123,
+            "count" : 100,
+            "type" : "VEC3"
+        },
+        {
+            "componentType" : 5123,
+            "count" : 100,
+            "type" : "VEC4"
+        },
+        {
+            "componentType" : 5123,
+            "count" : 100,
+            "type" : "VEC4"
+        },
+        {
+            "componentType" : 5123,
+            "count" : 100,
+            "type" : "SCALAR"
+        },
+        {
+            "componentType" : 5123,
+            "count" : 100,
+            "type" : "VEC3"
+        },
+        ...
+    ],
+    "bufferViews" : [
+        // Contains data for the first compressed animation.
+        {
+            "buffer" : 10,
+            "byteOffset" : 0,
+            "byteLength" : 10000
+        },
+        // Contains data for the second compressed animation.
+        {
+            "buffer" : 10,
+            "byteOffset" : 10000,
+            "byteLength" : 6000
+        },
+        ...
+    ]
+}
+
+
+```
+
+### JSON Schema
+
+For full details on the `Draco_animation_compression` extension properties, see the schema:
+
+* [extension property (the entirety of this specification is normative and included in Scope)](schema/node.DRACO_animation_compression.schema.json)
+
+## Conformance
+
+Below is the recommended process when a loader encounters a glTF asset with the Draco animation extension set:
+
+* If `Draco_animation_compression` is in `extensionsRequired` and the loader does not support the Draco animation extension, then the loader must fail loading the asset.
+* If the loader does not support the Draco animation extension and `Draco_animation_compression` is not in `extensionsRequired`, then load the glTF asset ignoring `Draco_animation_compression` in `extensions`.
+* If the loader does support the Draco animation extension, but will not process `Draco_animation_compression`, then the loader must load the glTF asset ignoring `Draco_animation_compression` in `extensions`.
+* If the loader does support the Draco animation extension, and will process `Draco_animation_compression` then:
+    * The loader must process `Draco_animation_compression` first. The loader must get the data from `Draco_animation_compression`'s `bufferView` property and decompress the data using a Draco animation decoder to a Draco animation.
+    * Then the loader must process `samplers` properties of the `animations`. When loading each element of `samplers`, you must ignore the `bufferView` and go to the previously decoded Draco animation from `compressedAnimations` to get the data of `input` and `outputs`. A loader must use the decompressed animation data to fill the `accessors` (e.g. [ThreeJS (non-normative)](https://github.com/FrankGalligan/three.js/blob/add_draco_animation_compression/examples/js/loaders/DRACOLoader.js)).
```

Comment:
To clarify — when "when loading each element of `samplers`" should probably say each element of `samplers` that references an input/output pair with corresponding compressed data. Assuming that an asset could contain a mix of compressed and uncompressed animation — or should that be disallowed?

Presumably input and output must always come from the same compressed animation?